### ISSUE TITLE
BUGS: instaling fresh new laravel template

### ISSUE THREAD

**Issue by YevheniiVolosiuk** [rcravens/kit#9]
OS: Windows 10 WSL 2 on Ubuntu 22.04

Stack Trace:
![image](https://github.com/user-attachments/assets/303292ae-dee0-48a8-a3ef-a2da87d53298)
![image](https://github.com/user-attachments/assets/60957584-831d-4232-bc96-85d27e93acc1)

**Comment by rcravens** [comment]
i think the root issue is this:

<img width="692" alt="image" src="https://github.com/user-attachments/assets/0b3f1d52-6317-4d49-ad87-59e7c91d6c18">

You need to provide it a github repo URL. something like : https://github.com/rcravens/laravel_demo.git

**Comment by rcravens** [comment]
This might be something that maybe only works on a mac: 

<img width="1068" alt="image" src="https://github.com/user-attachments/assets/5dfd7d9c-893c-4b69-a0ec-e85595e895da">


I'll need to look into that "open" command on WSL.

**Comment by YevheniiVolosiuk** [comment]
> i think the root issue is this:
> <img alt="image" width="692" src="https://private-user-images.githubusercontent.com/228847/364979868-0b3f1d52-6317-4d49-ad87-59e7c91d6c18.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1ODA1MzUsIm5iZiI6MTcyNTU4MDIzNSwicGF0aCI6Ii8yMjg4NDcvMzY0OTc5ODY4LTBiM2YxZDUyLTYzMTctNGQ0OS1hZDg3LTU5ZTdjOTFkNmMxOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQyMzUwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNjc4ODVhMGZkYWFiNWM5NzYzNjJjYmEyOTljZjVmYTg0MDIzNjNhMmIzNzA1Yzk5M2QyMThlMzllOTI1ODJmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.M5CNg48p-TGuAjDRZ9u6CRPdaReONqZp2hWzStL6hKI">
> 
> You need to provide it a github repo URL. something like : https://github.com/rcravens/laravel_demo.git

Yes, it is my fault I provided my name before and thought it would prompt one more time for a name, and at the end it downloaded a Laravel too code/ folder but bugs are all the same.
